WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -198,19 +198,10 @@ class NUXAbstractViewController: UIViewController, NUXSegueHandler, LoginWithLog
         return isCancellable() && navigationController?.viewControllers.first == self
     }
 
-
-    /// Checks if the signin vc modal should be cancellable. The controller is
-    /// cancellable when there is a default wpcom account, or at least one
-    /// self-hosted blog.
-    ///
-    /// - Returns: True if cancellable. False otherwise.
+    /// Returns true whenever the current ViewController can be dismissed.
     ///
-    @objc func isCancellable() -> Bool {
-        // if there is an existing blog, or an existing account return true.
-        let context = ContextManager.sharedInstance().mainContext
-        let blogService = BlogService(managedObjectContext: context)
-
-        return AccountHelper.isDotcomAvailable() || blogService.blogCountForAllAccounts() > 0
+    func isCancellable() -> Bool {
```

Comment:
Suggestion: I know this wasn't introduced in this PR but this could be good chance to rename this as `isDismissable`.

Reply:
Thanks Diego! adding that to my backlog, i'll be doing a major review / cleanup, once the WordPressAuthDelegate is fully implemented.